tools/expotools/src/check-packages/runPackageScriptAsync.ts

```diff
@@ -0,0 +1,44 @@
+import chalk from 'chalk';
+
+import { spawnAsync } from '../Utils';
+import { Package } from '../Packages';
+import logger from '../Logger';
+
+const { cyan, gray, red, reset } = chalk;
+
+/**
+ * Executes specified script name on given package.
```

Comment:
```suggestion
 * Executes the specified script (defined in package.json under "scripts") on the given package.
```

This explains more clearly which scripts are run.